source/includes/_locations.md

```diff
@@ -14,14 +14,16 @@ $api = new Api('<INSERT_API_KEY>', '<INSERT_API_SECRET>');
 $success = $api->post('/v1/clients-and-locations/locations/', [
     'name'                 => 'Le Bernardin',
     'url'                  => 'le-bernardin.com',
-	'business-category-id' =>  605,
-	'country'              => 'USA', // 3 letter iso code
-	'address1'             => '155 West 51st Street',
+    'business-category-id' =>  605,
+    'country'              => 'USA', // 3 letter iso code
+    'address1'             => '155 West 51st Street',
     'address2'             => '',
-	'region'               => 'NY' // State or Region
-	'city'                 => 'New York',
-	'postcode'             => '10019',
-    'telephone'            => '+1 212-554-1515'
+    'region'               => 'NY', // State or Region
+    'city'                 => 'New York',
+    'postcode'             => '10019',
+    'telephone'            => '+1 212-554-1515', 
+    'email_addresses' 	   => ['test@lebernardin.com'], 
```

Comment:
Please explain what these fields are for. Also, it looks like other fields use hyphens as separators. For consistency (if these are new fields) these should also use hyphens.